src/metabase/api/card.clj

```diff
@@ -943,7 +968,8 @@ saved later when it is ready."
   ([card      :- su/Map
     param-key :- su/NonBlankString
     query     :- (s/maybe su/NonBlankString)]
-   (let [param       (get (m/index-by :id (:parameters card)) param-key)
+   (let [params      (parameters card)
+         param       (get (m/index-by :id params) param-key)
```

Comment:
I think we should have a unit test for this where the params come from template tags